### PR TITLE
chore(docs): Update "Centralized Navigation"

### PR DESCRIPTION
**Pull request by polishedwp**
I am not sure what happens and where the original PR branch went, but this PR replaces #12562

### CODE REVIEW

**Comment by LekoArts** [comment]
Cool, thanks! Could you please also incorporate these comments?
https://github.com/gatsbyjs/gatsby/pull/12562/files/a5221513b3db8156d82b8862a3a3178188f3afbb#r278368132

File: docs/docs/centralizing-your-sites-navigation.md [hunk 1]
```diff
@@ -191,65 +191,67 @@ const Layout = ({ children }) => (
...
+            to="/"
+            style={{
+              color: 'white',
+              textDecoration: 'none',
+            }}
+          >
+            {siteTitle}
+          </Link>
+        </h1>
++        <div>
```
Comment (freiksenet) [inline docs/docs/centralizing-your-sites-navigation.md]: Seems that you've accidentally pasted it with the `+` symbols, could you remove them?

File: docs/docs/centralizing-your-sites-navigation.md [hunk 2]
```diff
@@ -191,65 +191,67 @@ const Layout = ({ children }) => (
...
++					<nav>
++	           <ul style={{ display: 'flex', flex: 1 }}>
++						{ menuLinks.map(link =>
++								<li key={link.name} style={{
++									'listStyleType': 'none',
++									padding: `1rem` }}>
++									<Link style={{ color: `white` }} to={link.link}>{link.name}</Link>
++							</li>)
++					}
++				</nav>
```
Comment (freiksenet) [inline docs/docs/centralizing-your-sites-navigation.md]: You are missing closing `</ul>` tag.

File: docs/docs/centralizing-your-sites-navigation.md [hunk 3]
```diff
@@ -191,65 +191,67 @@ const Layout = ({ children }) => (
 
 Locate the `header.js` file inside `src/components` and remove everything so only the functional component definition is left (everything else is just boilerplate code given to us when generating our project):
 
-```diff:title=src/components/header.js
+```diff:titlesrc/components/header.js
```
Comment (freiksenet) [inline docs/docs/centralizing-your-sites-navigation.md]:
```
diff:title=src/components/header.js
```

File: docs/docs/centralizing-your-sites-navigation.md (hunk 1 above)
Comment (LekoArts) [reply docs/docs/centralizing-your-sites-navigation.md]: It was a `diff` block, but I changed that as this code should be copy/paste really

**Comment by LekoArts** [comment]
Fixed the code example (some typos) and changed the diff to a normal code block (with highlighting) so that you can copy/paste better

**Pull request merged**